src/components/header/components/shared/Cart.component.tsx

```diff
@@ -0,0 +1,13 @@
+export const CartComponent = () => (
+    <a href={'/buy'}>
```

Comment:
`href={'/buy'}`

I think you can use all `href` like this `href='/buy'`, without extra `{}`

Reply:
done